src/logger.hpp

```diff
@@ -20,46 +30,66 @@ enum class LogLevel
 class Logger
```

Comment:
Yes, I know it, but in other case it will lose the functionality (like number, function name, file name).
I also can't edit the template Singleton because it doesn't have any sense.